test/uk/gov/hmrc/payetaxcalculatorfrontend/controllers/QuickCalcControllerSpec.scala

```diff
@@ -197,28 +194,30 @@ class QuickCalcControllerSpec() extends UnitSpec with Results with OneAppPerSuit
   }
 
   "Show Age Form" should {
-    "return 200, with current list of aggregate data containing Tax Code: 1150L and \"YES\" for is not Over65" in {
+    "return 200, with current list of aggregate data containing Tax Code: 1150L and \"YES\" is they are Over State Pension Age" in {
       val controller = new QuickCalcController(messages.messages, cacheReturnTaxCodeAndIsOverStatePension)
       val action = csrfAddToken(controller.showTaxCodeForm())
       val result = action.apply(request)
       val status = result.header.status
       val responseBody = contentAsString(result)
       val parseHtml = Jsoup.parse(responseBody)
```

Comment:
we seem to be repeating this pattern of extracting HTML source code in quite a few test cases so it might make sense to extract the lines starting with `csrfAddToken(...)` to a helper method and re use it wherever possible.